connectors/spark/src/test/java/io/unitycatalog/spark/BaseSparkIntegrationTest.java

```diff
@@ -67,8 +70,9 @@ protected SparkSession createSparkSessionWithCatalogs(boolean renewCred, String.
   @Override
   public void setUp() {
     super.setUp();
+    // Some Delta Spark functionalities needs testing mode to be turned on so that we can test.
+    System.setProperty("spark.testing", "true");
```

Comment:
Why we need to set this `spark.testing` property for every `setUp`?   What are the cases that we will use this ?

Reply:
Okay, got. it's from oss-delta.